src/plugins/mer/merbuildconfiguration.cpp

```diff
@@ -0,0 +1,143 @@
+/****************************************************************************
+**
+** Copyright (C) 2012 - 2019 Jolla Ltd.
+** Contact: http://jolla.com/
+**
+** This file is part of Qt Creator.
+**
+** GNU Lesser General Public License Usage
+** Alternatively, this file may be used under the terms of the GNU Lesser
+** General Public License version 2.1 as published by the Free Software
+** Foundation and appearing in the file LICENSE.LGPL included in the
+** packaging of this file.  Please review the following information to
+** ensure the GNU Lesser General Public License version 2.1 requirements
+** will be met: http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html.
+**
+** Other Usage
+**
+** Alternatively, this file may be used in accordance with the terms and
+** conditions contained in a signed written agreement between you and Digia.
+**
+****************************************************************************/
+
+#include "merbuildconfiguration.h"
+#include "merconstants.h"
+
+#include "qmakeprojectmanager/qmakeproject.h"
+#include "qmakeprojectmanager/qmakeprojectmanagerconstants.h"
+
+#include <utils/algorithm.h>
+#include <utils/qtcassert.h>
+
+#include <QTextStream>
+#include <QRegularExpression>
+#include <QDebug>
+
+using namespace Utils;
+using namespace ProjectExplorer;
+using namespace QmakeProjectManager;
+
+namespace {
+
+const char MER_VARIABLES_CACHE_FILENAME[] = "/.qmake_variables.cache";
```

Comment:
It should be obvious from the file name that it is a convention specific to our SDK. Given that `mb2` already creates the `.mb2` directory for its runtime files, `"/.mb2/qmake_variables.cache"` could be used.

I do not really like including the slash here - the code using it is shorter but less readable.